content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -0,0 +1,182 @@
+- Suggest improvements on [Slack](http://slack.k8s.io/) or the [SIG docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
+
+After [signing the CLA](/docs/contribute/new-content/overview/#sign-the-cla), anyone can also:
+
+- Open a pull request to improve existing content, add new content, or write a blog post or case study
+- Create diagrams, graphics assets, and embeddable screencasts and videos
+
+For more information, see [contributing new content](/docs/contribute/new-content/).
+
+## Members
+
+A member is someone who has submitted multiple pull requests to `kubernetes/website`. Members are a part of the [Kubernetes GitHub organization](https://github.com/kubernetes).
+
+Members can:
+
+- Do everything listed under [Anyone](#anyone)
+- Use the `/lgtm` comment to add the LGTM (looks good to me) label to a pull request
+- Use the `/hold` comment to block merging for a pull request
+- Use the `/assign` comment to assign a reviewer to a pull request
+- Provide non-binding review on pull requests
+- Use automation to triage and categorize issues
+- Document new features
+
+### Becoming a member
+
+After submitting at least 5 substantial pull requests and meeting the other [requirements](https://github.com/kubernetes/community/blob/master/community-membership.md#member):
+
+1.  Find two [reviewers](#reviewers) or [approvers](#approvers) to [sponsor](/docs/contribute/advanced#sponsor-a-new-contributor) your membership.
+
+    Ask for sponsorship in the [#sig-docs channel on Slack](https://kubernetes.slack.com) or on the
+    [SIG Docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
+
+    {{< note >}}
+    Don't send a direct email or Slack direct message to an individual
+    SIG Docs member. You must request sponsorship before submitting your application.
+    {{< /note >}}
+
+2.  Open a GitHub issue in the [`kubernetes/org`](https://github.com/kubernetes/org/) repository. Use the **Organization Membership Request** issue template.
+
+3.  Let your sponsors know about the GitHub issue. You can either:
+  - Mention their GitHub username them in an issue (`@<GitHub-username>`)
+  - Send them the issue link using Slack or email.
+
+  Sponsors will approve your request with a `+1` vote. Once your sponsors approve the request, a Kubernetes GitHub admin adds you as a member. Congratulations!
+
+  If your membership request is not accepted you will receive feedback. After addressing the feedback, apply again.
+
+4. Accept the invitation to the Kubernetes GitHub organization in your email account.
+
+  {{< note >}}
+  GitHub sends the invitation to the default email address in your account.
+  {{< /note >}}
+
+## Reviewers
+
+Reviewers are responsible for reviewing open pull requests. Unlike member feedback, you must address reviewer feedback. Reviewers are members of the [@kubernetes/sig-docs-{language}-reviews](https://github.com/orgs/kubernetes/teams?query=sig-docs) GitHub team.
```

Comment:
I like the idea of reviewer feedback being binding. 👍

Reply:
**Optionally** - explain conventions for how reviewers can leave non-binding feedback, eg “**Optionally**” :wink:

Reply:
+1 for the phrase "optionally", I personally find all the nit-picking a little grating because I'm overly sensitive :p